bin/main.ml

```diff
@@ -195,6 +195,24 @@ let all =
   ; Caching.command
   ]
 
+let common_commands_synopsis =
+  (* Short reminders for the most used and useful commands *)
+  let commands =
+    [ "build [--wait]"
```

Comment:
Should be `--watch` (and bellow as well)

Reply:
Doh! Thank you for catching that.

This confirms my thought that I should figure how how to make this testable...